### DISCUSSION TITLE
Navigation within Guide

### DISCUSSION THREAD

**Discussion by Azkel** [PoshCode/PowerShellPracticeAndStyle#179] [Q&A] [answered]
Hi, 
As I am trying to read the guide 'page by page', I find it a little bit irritating, that even though there is a Table of Contents, I have to go back to it to reach next section.

Maybe we could add at the bottom links to the "next section" and "previous section" based on Table o Contents, to make it easier to read? 

Best regards,
Michał

**Comment by Jaykul** [comment, marked as answer]
Have you tried [this view](https://poshcode.gitbooks.io/powershell-practice-and-style/) or [downloading](https://legacy.gitbook.com/book/poshcode/powershell-practice-and-style/details) the PDF or eBook versions?

We should clearly update the main ReadMe.md on GitHub with those links to make them easier to find. I thought I already had, but it looks like all I did was set the first url in the description...

**Reply by God-damnit-all** [reply]
@Jaykul Is the eBook / PDF version(s) not available anymore? If so, why not?

**Reply by Jaykul** [reply]
It is not. Gitbook made that feature a "premium" feature, behind a $65/mo paywall.

I guess we need to investigate the options available nowadays. Given they've taken away the primary reason we were using them, we should probably set up a static-content generator site instead of giving gitbooks.io any traffic. 

But to get a readable PDF, we'll probably have to redo layouts or indexes for some new tooling and set it up in a build.

**Comment by Azkel** [comment]
Oh, sorry I was only browsing it on Github repository. Never mind then ;)
